desktop/src/main/java/bisq/desktop/main/market/offerbook/OfferBookChartView.java

```diff
@@ -528,12 +525,25 @@ public void updateItem(final OfferListItem offerListItem, boolean empty) {
                                     setGraphic(null);
                                 }
                             }
+
+                            /**
+                             * Renders cell content, if it has a single value or a range.
+                             * Should not be called for empty cells
```

Comment:
Is this comment really needed? Maybe naming `renderCellContent()` is enough as it is private method in very specific context.